doc/faq.rst

```diff
@@ -0,0 +1,99 @@
+==========================
+Frequently Asked Questions
+==========================
+
+Below are some of PySB's frequently asked questions. If your question is
+not answered here, you can try our
+`Gitter channel <https://gitter.im/pysb/pysb>`_.
+For more general Python related questions, we recommend `Stack
+Overflow <https://www.stackoverflow.com>`_.
+
+General
+=======
+
+* Do you support Python 3?
+
+    **Yes, with one caveat, which will be fixed in the next release**. PySB
```

Comment:
Now that #309 has been merged, this whole answer can basically be replaced with "YES!"

Reply:
Updated FAQ on Python 3 support. I left a note in place about the previous reversible rule syntax and that it needs to be replaced for Python 3 support.